src/wmnds/components/inset-text/_properties.njk.json

```diff
@@ -0,0 +1,14 @@
+{
+  "wmndsInsetTextProps": [
+    {
+      "name": "html",
```

Comment:
Update to contentHTML as per btn/link component.
Also add a contentText param as users may get confused with typing HTML.

Again, see release branch and look at btn/link components.